### PR TITLE
Add a simple build and smoke workflow - and package

### LINKED ISSUE #21: Implement simple Smoke test of Spinup-NEMO
Build from `requirements.txt` and import a module.

### PR DESCRIPTION
**Pull request by ma595**
As the title suggests. 

Merge #7 first followed by this PR. 

Closes #21

### CODE REVIEW

File: tests/test_forecast.py
Comment (surbhigoel77) [inline tests/test_forecast.py:1]: We can a line of intro at the beginning of the file, explaining what it the purpose of the file.

**Review by surbhigoel77 — APPROVED** [review]
Left a comment for a minor add-on comment to the `test_forecast.py` file. The code runs fine. LGTM

File: tests/test_forecast.py
Comment (ma595) [reply tests/test_forecast.py:1]: Sounds good. I'll make the change.

**Comment by ma595** [comment]
> Left a comment for a minor add-on comment to the `test_forecast.py` file. The code runs fine. LGTM

As discussed, code does not run locally without explicitly setting `export PYTHONPATH=$(pwd)/lib:$PYTHONPATH.`

**Comment by ma595** [comment]
> > Left a comment for a minor add-on comment to the `test_forecast.py` file. The code runs fine. LGTM
> 
> As discussed, code does not run locally without explicitly setting `export PYTHONPATH=$(pwd)/lib:$PYTHONPATH.`

Fixed now by packaging code. Consider updating packaging with `poetry` or `pyproject.toml`

**Comment by ma595** [comment]
@isaacaka can you please review this?

**Pull request merged**